get_libwallet_api.sh

```diff
@@ -1,60 +1,66 @@
 #!/bin/bash
 
 
-BITMONERO_URL=https://github.com/monero-project/bitmonero.git
-BITMONERO_BRANCH=master
+MONERO_URL=https://github.com/monero-project/monero.git
+MONERO_BRANCH=master
 # thanks to SO: http://stackoverflow.com/a/20283965/4118915
 CPU_CORE_COUNT=$(grep -c ^processor /proc/cpuinfo 2>/dev/null || sysctl -n hw.ncpu)
 pushd $(pwd)
 ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 
 INSTALL_DIR=$ROOT_DIR/wallet
-BITMONERO_DIR=$ROOT_DIR/bitmonero
+MONERO_DIR=$ROOT_DIR/monero
 
 
-if [ ! -d $BITMONERO_DIR ]; then
-    git clone --depth=1 $BITMONERO_URL $BITMONERO_DIR --branch $BITMONERO_BRANCH --single-branch
+if [ ! -d $MONERO_DIR ]; then
+    git clone --depth=1 $MONERO_URL $MONERO_DIR --branch $MONERO_BRANCH --single-branch
 else
-    cd $BITMONERO_DIR;
-    git checkout $BITMONERO_BRANCH
+    cd $MONERO_DIR;
+    git checkout $MONERO_BRANCH
     git pull;
 fi
 
-echo "cleaning up existing bitmonero build dir, libs and includes"
-rm -fr $BITMONERO_DIR/build
-rm -fr $BITMONERO_DIR/lib
-rm -fr $BITMONERO_DIR/include
+echo "cleaning up existing monero build dir, libs and includes"
+rm -fr $MONERO_DIR/build
+rm -fr $MONERO_DIR/lib
+rm -fr $MONERO_DIR/include
 
-mkdir -p $BITMONERO_DIR/build/release
-pushd $BITMONERO_DIR/build/release
+mkdir -p $MONERO_DIR/build/release
+pushd $MONERO_DIR/build/release
 
 if [ "$(uname)" == "Darwin" ]; then
     # Do something under Mac OS X platform        
-    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D CMAKE_INSTALL_PREFIX="$BITMONERO_DIR"  ../..
+    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D INSTALL_VENDORED_LIBUNBOUND=ON -D CMAKE_INSTALL_PREFIX="$MONERO_DIR"  ../..
 elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
     # Do something under GNU/Linux platform
-    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D CMAKE_INSTALL_PREFIX="$BITMONERO_DIR"  ../..
+    PLATFORM="Linux"
+    cmake -D CMAKE_BUILD_TYPE=Release -D STATIC=ON -D BUILD_GUI_DEPS=ON -D CMAKE_INSTALL_PREFIX="$MONERO_DIR"  ../..
```

Comment:
Btw, just to clarify, when I said we should be using distro's packaged libraries, I didn't refer to what this script should do. Any reasonable packaging script for monero-core shouldn't be using this script at all (it should rely on a separate package for monerod and/or libwallet -- which is what Arch set of packages do already). Just wanted to clarify this, in case you have preferences regarding how you want this script to work -- I have zero input with regard to this script.